### ISSUE TITLE
Support several extensions in LanguageServerDefinition

### ISSUE THREAD

**Issue by nixel2007** [NipunaRanasinghe/lsp4intellij#8]
**Is your feature request related to a problem? Please describe.**

Old intellij-lsp supports several extensions while registring LanguageServerDefinition via semicolon separated string as first parameter. Current implementation supports only one extension (as discussed at #4)

**Describe the solution you'd like**

Old semicolon behavior is ok.

**Describe alternatives you've considered**

Array or list of strings is also ok :)

**Comment by NipunaRanasinghe** [comment]
@nixel2007 Please note that I've changed the API to input the extensions as a  **comma separated string**, since I thought using `,` as the the separator would be more intuitive. :D

**Comment by nixel2007** [comment]
@NipunaRanasinghe migrated to using of SPLIT_CHAR, thanks.

but... no luck. stil have strange URI for wrapper:

```
2019-02-26 12:53:57,453 [  10953]   INFO - ntellij.IntellijLanguageClient - Opened РаботаСКоннекторами.os 
2019-02-26 12:53:57,454 [  10954]   INFO - ntellij.IntellijLanguageClient - Instantiating wrapper for os : file:///D:D: 
2019-02-26 12:53:57,461 [  10961]   INFO - ntellij.IntellijLanguageClient - Adding file РаботаСКоннекторами.os 
2019-02-26 12:53:57,462 [  10962]  ERROR - plication.impl.ApplicationImpl - String index out of range: -1 
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(String.java:1967)
	at com.github.lsp4intellij.utils.FileUtils.sanitizeURI(FileUtils.java:135)
	at com.github.lsp4intellij.utils.FileUtils.editorToURIString(FileUtils.java:94)
	at com.github.lsp4intellij.client.languageserver.wrapper.LanguageServerWrapper.connect(LanguageServerWrapper.java:257)
	at com.github.lsp4intellij.IntellijLanguageClient.lambda$editorOpened$0(IntellijLanguageClient.java:148)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by nixel2007** [comment]
oh, @NipunaRanasinghe, sorry! i thought it was a fix for another issue #9 :) 
several extensions work!

**Comment by NipunaRanasinghe** [comment]
Great! I'll work on #9 and update you when it's fixed.

**Comment by NipunaRanasinghe** [comment]
@nixel2007 windows issue is fixed with #11. Could you please verify?? :)
